src/applications/appeals/995/containers/App.jsx

```diff
@@ -173,6 +174,18 @@ export const App = ({
     </RoutedSavableApp>
   );
 
+  // Add Datadog UX monitoring to the application
+  useBrowserMonitoring({
+    loggedIn,
+    formId: 'sc', // becomes "scBrowserMonitoringEnabled" feature flag
+    version: '1.0.0',
+    applicationId: '2779ccc3-be87-4b2d-a757-9ff54b58761b',
+    clientToken: 'pub442ae6e93be9f8d93a358bf78095c88a',
+    site: 'ddog-gov.com',
```

Comment:
not sure how things like these are handled here, but i usually like to store configurations/credentials like these in a separate file — not a blocker though, just wanted to call it out in case any of these are reusable 😄

Reply:
That's a good point! We could move the application ID and client token into the constants file. Also, they aren't used anywhere else. This also doesn't need the `site` setting since it's already a default value.